database/dynamodb.go

```diff
@@ -29,34 +22,56 @@ type AWSCredentials struct {
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
 
-	env, found := os.LookupEnv(Constants["ENV"])
-	if !found {
-		// load the env values using the `setUpEnv` function.
-		utils.SetUpEnv()
-	}
+	env := os.Getenv(utils.ENV)
```

Comment:
This line is trying to get the environment variable utils.ENV. However, the variable utils.ENV has not been defined yet. This means that the call to os.Getenv() will return the empty string.

Reply:
https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32/files#diff-ae79249906eadef6d2aa935b7a43f98abb5092136b4c421da80d2f7dc29e94cdR4 this line in the file Constants.go. @iamitprakash

Reply:
For the development env, we are storing the `.env` flie, for production need to store them in `template.yaml` file

Reply:
For production its not yet setup.